src/Serilog/Capturing/PropertyValueConverter.cs

```diff
@@ -42,7 +42,10 @@ partial class PropertyValueConverter : ILogEventPropertyFactory, ILogEventProper
             typeof(long), typeof(ulong), typeof(float), typeof(double), typeof(decimal),
             typeof(string),
             typeof(DateTime), typeof(DateTimeOffset), typeof(TimeSpan),
-            typeof(Guid), typeof(Uri)
+            typeof(Guid), typeof(Uri),
+            #if NET6_OR_GREATER
```

Comment:
Rather than use TFM-based conditionals, can we use a flag like `FEATURE_DATE_AND_TIME_ONLY` here?

We've stuck to this scheme for a while since it makes it a bit easier to tease apart what is enabled for each target.

Reply:
yep. will do it now

Reply:
done